package/gargoyle/files/www/templates/client_server_template

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Please remove leading blank line. If blank line separator needed, add on the parent page.
Please check all instances where you have called this template and add the blank line there to create proper separation (if/as needed).